.travis.yml

```diff
@@ -25,11 +25,19 @@ env:
     - "GEM=guides"
 rvm:
   - 2.2.4
-  - 2.3.0
-  - ruby-head
+  # - 2.3.0
+  # - ruby-head
 matrix:
+  include:
+    - rvm: jruby-9.0.5.0
+      jdk: oraclejdk8
+      env:
+        - "JRUBY_OPTS='-J-Xmx1024M' -Xcompat.version=2.0 --server --Xcompile.invokedynamic=false -Xcli.debug=true --debug"
+        # Test a random part of Rails. Something is better than nothing, eh?
+        - 'GEM=$(ruby -e "print %w[railties ap ac am amo as av aj ar:postgresql ar:mysql2 ar:sqlite3 aj:integration].sample")'
```

Comment:
one solution to reduce overhead but ensure it's run regularly, randomly select one gem to run against